sklearn/preprocessing/data.py

```diff
@@ -54,41 +58,82 @@
 )
 
 
-def _mean_and_std(X, axis=0, with_mean=True, with_std=True):
-    """Compute mean and std deviation for centering, scaling.
+def _incremental_mean_and_var(new_X, axis=0, last_mean=.0, last_var=None,
```

Comment:
Is the axis keyword required here (and in other methods) since it seems to be used only once with axis=0 and private anyway?